llvm/test/MC/AArch64/SVE2p2/fmmla-diagnostics.s

```diff
@@ -0,0 +1,9 @@
+// RUN: not llvm-mc -triple=aarch64 -show-encoding -mattr=+sve2p2,+f16mm 2>&1 < %s| FileCheck %s
+
+// --------------------------------------------------------------------------//
+// Invalid element width
+
```

Comment:
Nit:
We could add a test for the movprfx with predication.
The MOVPRFX must be unpredicated.
So this:

movprfx z0.d, p0/z, z6.d
fmmla z0.b, z2.b, z3.b